### ISSUE TITLE
Exploding gradients problem on the last batch in dev set

### ISSUE THREAD

**Issue by Iulian277** [cosmoquester/transformers-bart-pretrain#21]
In this current version of the code, there is a problem with the `dev-dataset`. After some training epochs,
depending on the `dev-batch-size` argument, the last `batch size` in the validation dataset could become very small
and the loss will be wrongly computed, because it will contain `NaN` values.

This leads to the **exploding gradients** problem where the computed gradients will have big values, hence
the **loss** will increase suddenly.
